java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -22,9 +22,11 @@ public interface ServerManagementClusterCommands {
      * @return Response from Redis cluster with a <code>Map{@literal <String, String>}</code> with
      *     each address as the key and its corresponding value is the information for the node.
      * @example
-     *     <p><code>
-     *     {@literal Map<String, String>} routedInfoResult = clusterClient.info().get().getMultiValue();
-     *     </code>
+     *     <pre>{@code
+     * ClusterValue<String> payload = clusterClient.info().get();
+     * assert payload.getMultiValue().get("node1.example.com:6379").contains("# Stats");
```

Comment:
this isn't a great example, because it creates the assumption that the user should know the names of their cluster nodes.

Reply:
Why does it exactly create that assumption one could always loop over a map if wanted? Alternatively we  could use for loops in the examples or streams to make assertions if that's the preference. (it could make the examples a bit more bloated)

Reply:
In this example you give the name of the node as a constant. Yes, the name could be gotten from elsewhere, but this is not what happens in this example.
Again, assume that the reader isn't familiar with the library and its assumptions - make everything explicit, explain your assumptions and intentions.